src/Http/Requests/Messages/ReadMessagesRequest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace RonasIT\Chat\Http\Requests\Messages;
+
+use RonasIT\Chat\Contracts\Requests\ReadMessagesRequestContract;
+use RonasIT\Chat\Contracts\Services\MessageServiceContract;
+use RonasIT\Support\BaseRequest;
+use RonasIT\Chat\Models\Message;
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+use Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException;
+
+class ReadMessagesRequest extends BaseRequest implements ReadMessagesRequestContract
+{
+    protected ?Message $message;
```

Comment:
please also rename other places too

```suggestion
    protected ?Message $fromMessage;
```